ansible.cfg

```diff
@@ -16,5 +16,5 @@ become = True
 
 [ssh_connection]
 ssh_args = -o ControlMaster=auto -o ControlPersist=900s
-control_path = %(directory)s/%%C
+control_path = %(directory)s/%%h-%%r
```

Comment:
In https://en.wikibooks.org/wiki/OpenSSH/Cookbook/Multiplexing it says: 

> Starting with 6.7, the combination of %r@%h:%p and variations on it can be replaced with %C which by itself generates a hash from the concatenation of %l%h%p%r. The advantage there is that the hash can be shorter than its combined elements while still uniquely identifying the connection and it obfuscates the connection information which would have been otherwise displayed in the socket's name.

The thing is that the path information with AWS on Mac gets to long, hence %C solves this problem, because it generates a unique hash. IMHO this is the most safe solution, than to shorten the available information... 

To the team, is there maybe another / better workaround, that works with all versions of ssh?